packages/lib/raqb/evaluateRaqbLogic.test.ts

```diff
@@ -15,8 +15,9 @@ describe("evaluateRaqbLogic", () => {
       queryValue: {
         id: "88bbb99b-cdef-4012-b456-71925b0e4b7f",
         type: "group",
-        children1: {
-          "b9aa9b89-0123-4456-b89a-b19264fe16af": {
+        children1: [
```

Comment:
This is the core breaking change - that seems to break a lot of things when testing forms in apps. Especially with complex logic